cmd/kar-controllers/app/options/options.go

```diff
@@ -30,6 +30,10 @@ type ServerOption struct {
 	Dispatcher	bool
 	AgentConfigs 	string
 	SecurePort	int
+	DynamicPriority	bool  // If DynamicPriority=true then no preemption is allowed by program logic
+	Preemption 		bool  // Preemption is not allowed under DynamicPriority
+	SendUpdate		bool  // Send QueueJobStatus Updatae() to Etcd.  Not essential for Controller functions
+	Demo	bool  // Demo: add delay after moving HeadOfLine job to eventQueue, before examine qjqueue again
```

Comment:
Can you provide some details on how `Demo` is used?  Is it for performance evaluation?

Reply:
If Demo=false, then all submitted jobs are dispatched very quickly until cluster resources are exhausted.  It is difficult to observe the differentiation of treatment to the jobs.  Setting Demo=true, an artificial delay of 10 seconds is added, after moving the HeadOfLine job to eventQueue for dispatching, and before the ScheduleNext thread examine the qjqueue again.  This allows the controller to act in slow motion, and allows users to observe the effects from HeadOfLine reservation and DynamicPriority policy more clearly.

Reply:
Can the observations be logged instead?  I'm wondering if this option is an end user option that we want to expose?  If it is for demo only purposes might be better to log the observed behavior.  For example, we could use Prometheus which is a common tool used with k8s logs to capture events from within a controller in order evaluate behavior.   Please share your thoughts.

Reply:
Per discussion, users will not have the slow motion demo functionality.  Demo option removed.

Reply:
Sound good.  Thanks!